src/Oro/Bundle/EmbeddedFormBundle/Resources/public/js/embed.form.js

```diff
@@ -69,6 +69,25 @@ var ORO = (function(ORO) {
             if (form) {
                 form.addEventListener('submit', this.onSubmit.bind(this));
             }
+
+            var scripts = this.container.querySelectorAll('script');
+            if (scripts.length > 0) {
+                var scriptsFragment = document.createDocumentFragment();
+                var realTag = null;
+                for (var k in scripts) if (scripts.hasOwnProperty(k)) {
```

Comment:
@scapital , please fix code style. 
1. Default value for `realTag` has no sense.
1. `platform/src/Oro/Bundle/EmbeddedFormBundle/Resources/public/js/embed.form.js: line 77, col 40, Expected '{' and instead saw 'if'.`
1.
```
For in statement without curly braces at platform/src/Oro/Bundle/EmbeddedFormBundle/Resources/public/js/embed.form.js :
    75 |                var scriptsFragment = document.createDocumentFragment();
    76 |                var realTag = null;
    77 |                for (var k in scripts) if (scripts.hasOwnProperty(k)) {
------------------------^
    78 |                    realTag = document.createElement('script');
    79 |

```

Has to be
```js
                var realTag;
                for (var k in scripts) {
                    if (!scripts.hasOwnProperty(k)) {
                        continue;
                    }
```